### PR TITLE
[ADD][9.0] hr contract operating unit

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
# HR Contract with Operating Units

This module introduces the following features:
- Adds the Operating Unit (OU) to the Employee Contract.
- Security rules are defined to ensure that users can only see the Contracts of that Operating Units in which they are allowed access to.

### CODE REVIEW

File: hr_contract_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+================================
+
+This module introduces the following features:
+
+* Adds the Operating Unit (OU) to the Employee Contract.
+
+* Security rules are defined to ensure that users can only see the Contracts of that Operating Units in which they are allowed access to.
+
+
+Installation
```
Comment (elicoidal) [inline hr_contract_operating_unit/README.rst]: Remove the following 2 sections if empty

File: hr_contract_operating_unit/security/hr_contract_security.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
```
Comment (elicoidal) [inline hr_contract_operating_unit/security/hr_contract_security.xml]: change to `<odoo>`

**Comment by elicoidal** [comment]
The commit message is misleading (this is not expense but contract model).

Other than that :+1: !
Thanks for the contribution

**Review by elicoidal — CHANGES_REQUESTED** [review]
Commit message is misleading but overall :+1:

**Comment by AaronHForgeFlow** [comment]
I apologize for the commit message. Thank you @elicoidal

File: hr_contract_operating_unit/security/hr_contract_security.xml (hunk 2 above)
Comment (AaronHForgeFlow) [reply hr_contract_operating_unit/security/hr_contract_security.xml]: Should I do the same in the views files?

**Comment by AaronHForgeFlow** [comment]
@elicoidal can you please review again?

File: hr_contract_operating_unit/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   :alt: License: LGPL-3
+
+================================
+HR Contract with Operating Units
+================================
+
+This module introduces the following features:
+
+* Adds the Operating Unit (OU) to the Employee Contract.
+
```
Comment (elicoidal) [inline hr_contract_operating_unit/README.rst:12]: remove the empty line

File: hr_contract_operating_unit/README.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* Security rules are defined to ensure that users can only see the Contracts of that Operating Units in which they are allowed access to.
```
Comment (elicoidal) [inline hr_contract_operating_unit/README.rst]: non blocking but better to stick to 80 char lines

**Review by elicoidal — APPROVED** [review]
:+1:

File: hr_contract_operating_unit/README.rst (hunk 3 above)
Comment (AaronHForgeFlow) [reply hr_contract_operating_unit/README.rst:12]: Done. Thank you @elicoidal

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by serpentcs-dev1 — APPROVED** [review]
code review LGTM :+1:

**Pull request merged**